src/dashboard/src/fpr/utils.py

```diff
@@ -102,7 +102,7 @@ def get_revision_descendants(model, uuid, decendants):
     revision = model.objects.get(uuid=uuid)
     try:
         descendant = model.objects.get(replaces=revision)
-    except model.DoesNotExist:
+    except (model.DoesNotExist, ValidationError):
```

Comment:
Is this necessary?

Reply:
Thanks you for all your suggestions and will make those changes accordingly.